aws-predictions/src/main/java/com/amplifyframework/predictions/aws/NetworkPolicy.java

```diff
@@ -22,12 +22,6 @@
  * perform a Predictions operation.
  */
 public enum NetworkPolicy {
-    /**
-     * Only use cloud resources to perform online Predictions
-     * operation.
-     */
-    ONLINE("online"),
```

Comment:
Hm. How come?

Reply:
We still give them the implicit option of doing so (i.e. choose "auto" and just not add any offline models). iOS currently doesn't support "online" only mode and there was apparently already a discussion that decided against giving the customer an option to explicitly specify online-only mode.